ext/scheduler/airflow/airflow.go

```diff
@@ -87,8 +87,10 @@ func (s *Scheduler) DeployJobs(ctx context.Context, tenant tenant.Tenant, jobs [
 	}
 	defer bucket.Close()
 
-	bucket.WriteAll(spanCtx, filepath.Join(jobsDir, baseLibFileName), SharedLib, nil)
-
+	err = bucket.WriteAll(spanCtx, filepath.Join(jobsDir, baseLibFileName), SharedLib, nil)
+	if err != nil {
+		return err
```

Comment:
Wrap in meaning msg from optimus, It will only return the library error.

Reply:
Done